pkgs/applications/radio/direwolf/default.nix

```diff
@@ -29,7 +30,8 @@ stdenv.mkDerivation rec {
 
   strictDeps = true;
 
-  buildInputs = lib.optionals stdenv.isLinux [ alsa-lib udev ]
+  buildInputs = [ portaudio ]
```

Comment:
In looking at https://github.com/wb2osz/direwolf/blob/7d3c1d100ea38605bf5d11459f5a678be99fffab/CMakeLists.txt#L268-L292 It seems like `portaudio` is a darwin only requirement. Would you mind setting it as optional? `lib.optionals stdenv.isDarwin [ portaudio ]`

Reply:
Good catch by you as well! Present in latest commit.